framework/src/Volo.Abp.MongoDB/Volo/Abp/Domain/Repositories/MongoDB/MongoDbRepository.cs

```diff
@@ -81,6 +85,34 @@ public MongoDbRepository(IMongoDbContextProvider<TMongoDbContext> dbContextProvi
             return entity;
         }
 
+        public override async Task InsertManyAsync(IEnumerable<TEntity> entities, bool autoSave = false, CancellationToken cancellationToken = default)
+        {
+            foreach (var entity in entities)
+            {
+                await ApplyAbpConceptsForAddedEntityAsync(entity);
+            }
+
+            if (BulkOperationProvider != null)
+            {
+                await BulkOperationProvider.InsertManyAsync(this, entities, autoSave, cancellationToken);
```

Comment:
hi

The `MongoDbRepository` need to use `SessionHandle`, we should pass it or `IMongoDbContextProvider` to `BulkOperationProvider`,